### ISSUE TITLE
NSS Metric/Loss During Test

### ISSUE THREAD

**Issue by baharay** [samyak0210/saliency#23]
Hi,

Thank you very much for the great work and for publishing the code. I have a question about the ground truth for the NSS metric. 

I have evaluated your SALICON pre-trained models with the given commands. I can achieve your reported results except NSS metric. I get around 1.69 for the validation set while it is stated as 1.93 in the paper.  I recognized that in test.py at line 100, you use "gt" for calculating the NSS loss. I suppose "gt" is the continuous saliency map and "fixations" is the binary fixation map. After I change "gt" to "fixations" in this line, I could get an NSS score of 1.927. 

I would like to confirm this approach with you.

Best wishes,
Bahar

**Comment by samyak0210** [comment]
Hi,

Yes, it should have been "fixations" instead of "gt". I will make the necessary changes. Thank you for pointing out the mistake.

**Comment by baharay** [comment]
Thanks!
